public/static/app/perf-bb/PerfBBOutliersCtrl.js

```diff
@@ -0,0 +1,269 @@
+mciModule.controller('PerfBBOutliersCtrl', function(
+    $scope, $window, EvgUiGridUtil, EvgUtil, FORMAT, MDBQueryAdaptor, uiGridConstants,
+    uiGridGroupingConstants, STITCH_CONFIG, Stitch,
+) {
+    // Perf Failures View-Model.
+    const vm = this;
+    const project = window.project;
+    const LIMIT = 500;
+
+    const OUTLIERS_TYPE = {
+        DETECTED: 'detected',
+        SUSPICIOUS: 'suspicious',
+    };
+
+    let getCol;
```

Comment:
I prefer let over var because hoisting tends to be confusing and can led to difficult to debug errors. If you (or the evergreen team) feel strongly about it, I can switch.